src/navigator/Navigator.tsx

```diff
@@ -255,6 +256,11 @@ const sendScreens = (Navigator: typeof Stack) => (
       component={SendEnterAmount}
       options={noHeader}
     />
+    <Navigator.Screen
```

Comment:
related to my comment about the jumpstart flow handling this intro screen, i'm not sure that we should have a screen...i know the Earn info screen uses this pattern, but i'm not sure that it is the best one